src/neuroglancer/viewer.ts

```diff
@@ -59,12 +59,12 @@ import {RPC} from 'neuroglancer/worker_rpc';
 
 declare var NEUROGLANCER_OVERRIDE_DEFAULT_VIEWER_OPTIONS: any
 
-require('./viewer.css');
-require('neuroglancer/noselect.css');
-require('neuroglancer/ui/button.css');
+import './viewer.css';
+import'neuroglancer/noselect.css';
+import 'neuroglancer/ui/button.css';
 
 export class DataManagementContext extends RefCounted {
-  worker = new Worker('chunk_worker.bundle.js');
+  worker = new Worker('/chunk_worker.bundle.js');
```

Comment:
I agree it is important to be able to control the path here, but the python integration code, and various other users of neuroglancer, do rely on the relative path behavior.  Perhaps make this an option that can be configured either at build time or as a parameter to DataManagementContext?  One way to do that is to add:

declare var NEUROGLANCER_CHUNK_WORKER_BUNDLE_URL : string|undefined;

Then you can check if it is defined.

Reply:
Adding an environment variable seems like a reasonable option. I will get that set up.